### DISCUSSION TITLE
Questions about Legacy API

### DISCUSSION THREAD

**Discussion by kennytrumptech** [invertase/react-native-firebase#7847] [Q&A] [answered]
I recently received an email from Google Firebase saying that the Legacy API will be discontinued on June 20, 2024. However I am using react-native-firebase v18.7.3 and firebase-admin v12.0.0; not sure if I need to update? Thanks for your help!

**Comment by alexandrumogic** [comment, marked as answer]
Your version should include the correct APIs with JS SDK v9. I think this was already answered here: https://github.com/invertase/react-native-firebase/discussions/7197

**Reply by kennytrumptech** [reply]
@alexandrumogic thank you very much
